bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/testcases/ImportPdfImageTests.kt

```diff
@@ -0,0 +1,91 @@
+package net.gini.android.bank.sdk.exampleapp.ui.testcases
+
+import android.Manifest
+import androidx.test.espresso.Espresso.pressBack
+import androidx.test.espresso.IdlingRegistry
+import androidx.test.ext.junit.rules.activityScenarioRule
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.rule.GrantPermissionRule
+import androidx.test.uiautomator.UiDevice
+import net.gini.android.bank.sdk.exampleapp.ui.MainActivity
+import net.gini.android.bank.sdk.exampleapp.ui.resources.ImageUploader
+import net.gini.android.bank.sdk.exampleapp.ui.resources.PdfUploader
+import net.gini.android.bank.sdk.exampleapp.ui.resources.SimpleIdlingResource
+import net.gini.android.bank.sdk.exampleapp.ui.screens.CaptureScreen
+import net.gini.android.bank.sdk.exampleapp.ui.screens.ConfigurationScreen
+import net.gini.android.bank.sdk.exampleapp.ui.screens.ExtractionScreen
+import net.gini.android.bank.sdk.exampleapp.ui.screens.MainScreen
+import net.gini.android.bank.sdk.exampleapp.ui.screens.OnboardingScreen
+import net.gini.android.bank.sdk.exampleapp.ui.screens.ReviewScreen
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+/**
+ * Test class for flash on/off on CaptureScreen.
+ *
+ * Jira link for test case: [https://ginis.atlassian.net/browse/PM-21](https://ginis.atlassian.net/browse/PM-21)
+ *
+ * No automation for step 4 and 5, since it is not part of our SDK
+ */
+class ImportPdfImageTests {
+    @get:Rule
+    val activityRule = activityScenarioRule<MainActivity>()
+
+    @get: Rule
+    val grantPermissionRule: GrantPermissionRule =
+        GrantPermissionRule.grant(Manifest.permission.CAMERA)
```

Comment:
[Clarification] Do we still need the `Camera` permission? From what I see, we're uploading files from local storage.

Reply:
Yes we can do it.